erpnext/projects/doctype/timesheet/timesheet.js

```diff
@@ -20,16 +20,27 @@ frappe.ui.form.on("Timesheet", {
 					'project': child.project,
 					'status': ["!=", "Closed"]
 				}
-			}
-		}
+			};
+		};
 
 		frm.fields_dict['time_logs'].grid.get_field('project').get_query = function() {
 			return{
 				filters: {
 					'company': frm.doc.company
 				}
+			};
+		};
+
+		// set employee (and company) to the one that's currently logged in
```

Comment:
this will only be called the first time. If you want to add it to any new document, add it to `onload` or `refresh` trigger.

Reply:
I couldn't find an example where this doesn't work as expected. Do you have one?